x-pack/plugins/observability/public/hooks/slo/delete_slo.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Not super happy with this function yet. But I think there's more gains in moving to React Query than investing more time to optimize this.